baccmod/grid3d_acceptance_map_creator.py

```diff
@@ -252,12 +258,17 @@ def create_acceptance_map(self, observations: Observations) -> Background3D:
         elif self.method == 'fit':
             logger.info(f"Performing the background fit using {self.fit_fnc}.")
             corrected_counts = np.empty(count_background.shape)
+            self.sq_rel_residuals = {'mean': [], 'std': []}
             for e in range(count_background.shape[0]):
                 logger.info(f"Energy bin : [{self.energy_axis.edges[e]:.2f},{self.energy_axis.edges[e + 1]:.2f}]")
                 corrected_counts[e] = self.fit_background(count_background[e].astype(int),
                                                           exp_map_background_total_downsample.data[e],
                                                           exp_map_background_downsample.data[e],
                                                           )
+
+            logger.info("Average event counts diff/sqrt(fit) for each energies : "
+                        f"{np.round(self.sq_rel_residuals['mean'], 2)}\n" +
+                        f"Std = {np.round(self.sq_rel_residuals['std'], 2)}")
```

Comment:
Did you check behavior of this line with a restrictive log level as the tables will be empty and even if not displayed the line will be executed.